test/testdata/api/README.md

```diff
@@ -0,0 +1,14 @@
+To generate a valid certificate and private key use the following command:
+
+```bash
+# In OpenSSL ≥ 1.1.1
+openssl req -x509 -newkey rsa:4096 -sha256 -days 9999 -nodes \
+  -keyout key.pem -out cert.pem -subj "/CN=localhost" \
+  -addext "subjectAltName=DNS:localhost,IP:127.0.0.1,IP:::1"
+```
+
+To check a certificate use the following command:
+
+```bash
+openssl x509 -in cert.pem -text
+```
```

Comment:
Missing empty end of file ^^

Reply:
Do you mean I should remove the newline at the end of the file? I did that now.

Reply:
No actually you should add a new empty line if none. Not really important I know, but my kingdom my rules hehe

Reply:
Oh okay, there already was a newline afaik - maybe GitHub doesn't show it properly, re-adding... 😅